LunchTime/Restaurants/Freeland.cs

```diff
@@ -4,10 +4,11 @@
 using System.Collections.Generic;
 using System.Linq;
 using GeoCoordinatePortable;
+using LunchTime.Restaurants.MenuBrno;
 
 namespace LunchTime.Restaurants
 {
-    public class Freeland : RestaurantBase
+    public class Freeland : RestaurantBase, IRestaurant
```

Comment:
Wouldn't be better to apply the interface to base class?

Reply:
discussed: Interfaces are used for customizable registration.